ignite/pkg/cosmostxcollector/mocks/saver.go

```diff
@@ -5,9 +5,8 @@ package mocks
 import (
 	context "context"
 
-	mock "github.com/stretchr/testify/mock"
-
 	cosmosclient "github.com/ignite/cli/ignite/pkg/cosmosclient"
+	mock "github.com/stretchr/testify/mock"
```

Comment:
Import should go back to how it was before.

Reply:
indeed [651e399](https://github.com/ignite/cli/pull/3060/commits/651e3992b97dc0165ad459680ef429e195247a97)

I think the go format action will do it anyway :)